ytsync/setup.go

```diff
@@ -44,17 +44,17 @@ func (s *Sync) walletSetup() error {
 		return nil
 	}
 
-	numPublished, err := s.daemon.NumClaimsInChannel(s.LbryChannelName)
-	if err != nil {
-		return err
-	}
+	s.videosMapMux.Lock()
+	numPublished := uint64(len(s.syncedVideos)) //should we only count published videos? Credits are allocated even for failed ones...
+	s.videosMapMux.Unlock()
 	log.Debugf("We already published %d videos", numPublished)
 
 	if float64(numOnSource)-float64(numPublished) > float64(s.Manager.VideosLimit) {
```

Comment:
why are all these cast to floats? they can be ints here

Reply:
no clue. but you're completely right. Will fix